sql/2020/17_JAMstack/ssg_core_web_vitals_distribution.sql

```diff
@@ -0,0 +1,65 @@
+#standardSQL
+# Core Web Vitals distribution by SSG
```

Comment:
Is this identical to cdn_core_web_vitals_distribution.sql?

Reply:
yup, I merged them into 1 query.